website/src/content/help-needed.mdx

```diff
@@ -49,13 +49,19 @@ If you are interested in more advanced functionalities, let us know and open a [
 
 # Using a test or staging environment for my Custom Application
 
-During the development cycle of new features for a Custom Application, it can be helpful to showcase or test those new features before publishing them to the production environment.
+During the development cycle of new features for a Custom Application, it can be helpful to showcase or test those new features before publishing them to the production environment. This can be accomplished using [deployment previews](./concepts/deployment-previews) which allow you to inspect different versions of a Custom Application.
```

Comment:
```suggestion
During the development cycle of new features for a Custom Application, it can be helpful to showcase or test those new features before publishing them to the production environment. This can be accomplished using [deployment previews](/concepts/deployment-previews) which allow you to inspect different versions of a Custom Application.
```